docs/admin.rst

```diff
@@ -49,7 +54,24 @@ By default, the history log displays one line per change containing
 
 You can add other columns (for example the object's status to see
 how it evolved) by adding a ``history_list_display`` array of fields to the
-admin class
+admin class.
+
+.. code-block:: python
+
+    from django.contrib import admin
+    from simple_history.admin import SimpleHistoryAdmin
+    from .models import Poll, Choice
+
+    class PollHistoryAdmin(SimpleHistoryAdmin):
+        history_list_display = ["status"]
+        list_display = ["id", "name", "status"]
+        search_fields  = ['name', 'user__username']
+        list_per_page = 100
+
+    admin.site.register(Poll, PollHistoryAdmin)
+    admin.site.register(Choice, SimpleHistoryAdmin)
```

Comment:
It looks like you duplicated the code block below while adding `list_per_page = 100` to the example... Am I correct? If so, please remove the duplicated code 🙂